app/js/components/pages/Job.jsx

```diff
@@ -8,7 +8,7 @@ const Job = ({ plugin, workflow, inputArtifacts, submitJob, cancelJob }) => {
             <div className="page-header">
                 <h1>{plugin.name}: {workflow.description}</h1>
             </div>
-            <form>
+            <form onSubmit={(e) => { e.preventDefault(); submitJob(workflow); }}>
```

Comment:
This should actually definitely work, because `querySelector('form')` is really just returning the same DOM node as what `e` is bound to.